src/features/processPicker.ts

```diff
@@ -246,4 +333,16 @@ function execChildProcess(process: string, workingDirectory: string): Promise<st
             resolve(stdout);
         });
     });
+}
+
+function execChildProcessWithTimeout(process: string, workingDirectory: string): Promise<string> {
+    return new Promise<string>((resolve, reject) => {
+        child_process.exec(process, { timeout: 3000, cwd: workingDirectory, maxBuffer: 500 * 1024 }, (error: Error, stdout: string, stderr: string) => {
```

Comment:
I am not sure. I used execChildProcess as an example of calling exec on a child process.